packages/babel-plugin-proposal-pipeline-operator/test/fixtures/pipeline-operator/async-arrow/output.js

```diff
@@ -1,12 +0,0 @@
-var _ref, _;
```

Comment:
It looks like for this one, it doesn't throw so the input file is still there and thus expects an output file? https://github.com/babel/babel/tree/9283efaeb7a8c5b505b19332c3407dc44c668777/packages/babel-plugin-proposal-pipeline-operator/test/fixtures/pipeline-operator/async-arrow

Reply:
Ugh yeah I just missed that the test didn't re-run because it was skipped on Node 6.